ref/general/authorization.adoc

```diff
@@ -15,17 +15,18 @@
+
+The roles that users and groups are checked against can either be defined directly in the application or applied to the application by way of the server configuration. It is also possible to avoid defining roles altogether if you specify role names that match the group names in the user registry exactly. In this case, the server  maps the role to the group with a matching name by default. In this configuration, you do not specify an `<application-bnd>` element in the `server.xml` file.
 
-The roles that users and groups are checked against can either be defined directly in the application or applied to the application by way of the server configuration.
 Roles are defined directly in the application by setting the `@DeclareRoles` annotation or by using `<security-role>` elements in the `web.xml` deployment descriptor file.
 To apply roles to an application from the server configuration, specify the `<security-role>` element within the `<application-bnd>` element in the `server.xml` file.
 
-It is also possible to avoid defining roles altogether if you specify role names that match the group names in the user registry exactly. In this case, the server automatically maps the role to the group with a matching name. In this configuration, you do not specify an `<application-bnd>` element in the `server.xml` file. 
+In some use cases, an application needs to know only whether a user is authenticated or not. In such cases, configuration can be simplified by specifying `"**"` as the role name in the `@RolesAllowed` annotation or XML configuration. This value denotes that any authenticated user can access a resource. It is also possible to effectively bypass authorization altogether by using the `@PermitAll` annotation, which specifies that a resource is accessible to everyone, even users who are not authenticated.
 
-In some use cases, an application needs to know only whether a user is authenticated or not. In such cases, configuration can be simplified by using the `"**"` special value in the `@RolesAllowed` annotation or XML configuration. This value denotes that any authenticated user can access a resource. It is also possible to effectively bypass authorization altogether by using the `@PermitAll` annotation, which specifies that a resource is accessible to everyone, even users who are not authenticated.
+Typically, a developer can specify which roles are allowed to access an application resource directly in the source code by setting the `@RolesAllowed` annotation to contain the permissible roles. When a user is authenticated, the access groups to which the user is assigned are retrieved from a user registry or single sign-on (SSO) provider and added to the `Subject` object for that user. The Open Liberty server maps these groups to the roles that are defined in the application. If the role or roles that are mapped to the user match the roles that are specified for the resource, the user is granted access.
```

Comment:
This feels like it is describing internal implementation detail that really isn't relevant. I don't think we need to talk about a Subject her and the fact that we store user and group info in the Subject for role authorization is an implementation detail.